webapp/components/Badges.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div :class="[badges.length === 2 && 'hc-badges-dual']" class="hc-badges">
     <div v-for="badge in badges" :key="badge.id" class="hc-badge-container">
-      <img :title="badge.key" :src="badge.icon | proxyApiUrl" class="hc-badge" />
+      <img :title="badge.key" :src="`/img/badges/${badge.id}.svg`" class="hc-badge" />
```

Comment:
I decided to move the image files into webapp docker image. Why were they in backend image anyways? :thinking:

Reply:
that's a good question @roschaefer ...
could we just upload these to `Spaces` as well? then others could more easily reuse while customizing the badges, no? Also, it means smaller docker images

Reply:
The badges are small .svg files. At the moment we don't have a feature to add more badges. So I decided to put them into the docker image.